app/components/candidate_interface/reopen_banner_component.rb

```diff
@@ -14,13 +14,11 @@ def render?
 private
 
   def show_apply_1_reopen_banner?
-    apply_1? &&
-      CycleTimetable.between_cycles_apply_1?
+    apply_1? && CycleTimetable.between_cycles_apply_1?
```

Comment:
Do we need to take account of whether an application form was carried over or not  (whether it was successful or unsuccessful)?

Reply:
So this only shows renders on the application form show and review page. So they will only see it if they've already carried over.

Unsubmitted -> interstitial -> show page with banner 

unsuccessful -> dashboard -> carry over -> show page with banner